### DISCUSSION TITLE
Panel-like ScrollView

### DISCUSSION THREAD

**Discussion by JosephTLyons** [Textualize/textual#79] [Q&A]
Is there a way to add a border and a title to a `ScrollView` or any way to make a ScrollView look like a Rich Panel with scrolling content?

**Comment by CyberSmash** [comment]
I too am looking for this. Did you ever find a solution?

**Comment by JosephTLyons** [comment]
I wasn't able to come up with a solution for this, its possible I missed something, but at the time, I feel like it wasn't possible?  Might have to wait until Will responds to find out definitively.

**Comment by CyberSmash** [comment]
Hm yeah, as I'm looking deeper into this it appears that the `ScrollView` class takes a `style` parameter but it's never used for anything. Additionally, as the `ScrollView` is a `View` which renders a layout. Not sure how to properly override any of that. 

I currently have a `TreeControl` inside of a `ScrollView` and the best I've been able to do is subclass the `TreeControl` and override the `render` function to return `Panel(super().render())`. The problem with this is that it puts the boarder around the `TreeControl`, which means the bottom of the boarder is "off screen", and you have to scroll down to see it.

Is there a way to have some sort of a "Box" widget that passes all of its interactions to any sub-widgets? If so, we could easily whip together putting a `TreeControl` inside a `ScrollView` inside a `Box` widget. Speaking of...I wonder how the ScrollView passes its interactions to the TreeView...
